functions.php

```diff
@@ -78,3 +78,63 @@ function displayGuitars(array $guitarsArray):string {
     }
     return $result;
     }
+
+function addNewGuitar(\PDO $database):string {
+
+    $message = '<p>I want to add a new guitar for you!</p>';
+    $brandInDB = checkBrands($database);
```

Comment:
Functions calling other functions was the problem that Cat had when Charlie reviewed her code and it required a major rewrite